src/app/client/projects/desktop/src/app/modules/offline/components/content-manager/content-manager.component.ts

```diff
@@ -25,6 +25,10 @@ export class ContentManagerComponent implements OnInit, OnDestroy {
   apiCallTimer = timer(1000, 3000).pipe(filter(data => !data || (this.callContentList)));
   apiCallSubject = new Subject();
   completedCount: number;
+  noSpaceContentList: any;
+  previousList: any;
```

Comment:
only two property is required.
remove noSpaceContentList
and rename 
listToShow  > unHandledFailedList 
previousList  > handledFailedList